### ISSUE TITLE
Oligonucleotide hibridization support for mismatches

### ISSUE THREAD

**Issue by dgruano** [manulera/ShareYourCloning_backend#199]
I cannot think of a real-world example where this could be actually needed, but given the similarity with oligos annealing for a PCR, I think we may want to support this.

**Comment by manulera** [comment]
I don't think we should support this. The PCR makes sense, because the excess of oligo vs. template means that in the final product, the predominant sequence will overwhelmingly be the one from the primers. In a case like this you would have an actual mismatch inside the sequence.

I think, at least for now, that we can keep not supporting this use-case.
